website2/website/README.md

```diff
@@ -16,6 +16,11 @@
     specific language governing permissions and limitations
     under the License.
 -->
+ perl -pi -e 's/^  */\t/' Makefile
```

Comment:
What is the significance of these few lines?

Reply:
We should also remove the reference to version `0.20.2`

Reply:
okay